### ISSUE TITLE
Angle brackets and linkify

### ISSUE THREAD

**Issue by scottanderson42** [mozilla/bleach#128]
This:

```
html = '<http://moo.com>'
html = bleach.linkify(html)
```

Results in this:

```
&lt;http: &lt;a="" href="&amp;lt;a href=" http:="" moo.com"="" rel="nofollow"&gt;http://moo.com" rel="nofollow"&gt;<a rel="nofollow" href="http://moo.com">moo.com</a>=""&gt;
```

**Comment by willkg** [comment]
@lolsborn The plain-text email gist isn't available anymore. Can you provide it as a comment?

Also, can you retry this with Bleach 2.0 or later?

**Comment by lolsborn** [comment]
My username changed breaking the link.
https://gist.github.com/lolsborn/1bda622409ef321af1a9
```
--- FastMail Support Ticket ---
This is an automated email generated by the FastMail support
ticket system for support ticket ABCD1234.

DO NOT REPLY TO THIS EMAIL. REPLIES WILL NOT BE MONITORED.

To view the status of this ticket at any time, or to add an update, please use the link below in your web browser.

Ticket Link: <https://www.fastmail.fm/html/?MSignal=TZ-**ABCD123>

--- Ticket Update  (by first.last@gmail.com) (date 2014-07-30 18:25:59 UTC) (id ABCD123) ---
blah blah blah
I recently created an account to with email last.first@example.com and my work email first@yourco.com, but I somehow managed to save the wrong password in my blah blah blah
--- End Update ---
```

**Comment by willkg** [comment]
The original issue looks like this with master tip:

```
$ python
Python 3.6.5 (default, Apr  1 2018, 05:46:30) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import bleach
>>> bleach.linkify('<http://moo.com>')
'&lt;<a href="http://moo.com" rel="nofollow">http://moo.com</a>&gt;'
```

I think that looks good. Master tip will get released as Bleach 3.0.0 soon, so I'll add this to the milestone and close it out.
